src/compiler/crystal/compiler.cr

```diff
@@ -18,6 +18,7 @@ module Crystal
   # optionally generates an executable.
   class Compiler
     CC = ENV["CC"]? || "cc"
+    CL = "cl"
```

Comment:
I suggest renaming `CC` and `CL` to something like `POSIX_LINKER` and `WINDOWS_LINKER`